project/AppDependencies.scala

```diff
@@ -2,26 +2,21 @@ import sbt.*
 
 object AppDependencies {
 
-  private lazy val mongoHmrcVersion     = "1.6.0"
-  private lazy val bootstrapPlayVersion = "7.23.0"
+  private lazy val mongoHmrcVersion     = "1.7.0"
+  private lazy val bootstrapPlayVersion = "8.4.0"
 
   private lazy val compile: Seq[ModuleID] = Seq(
-    play.sbt.PlayImport.ws,
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"            % mongoHmrcVersion,
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping" % "1.13.0-play-28",
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"    % bootstrapPlayVersion,
-    "uk.gov.hmrc"       %% "play-frontend-hmrc"            % "7.29.0-play-28"
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"         % mongoHmrcVersion,
+    "uk.gov.hmrc"       %% "bootstrap-frontend-play-30" % bootstrapPlayVersion,
+    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "8.4.0"
```

Comment:
The update for this to 8.5.0 has been released since, so would be good to include this now in preparation for Tudor crown deployment